cmd/server/assets/email/default_invitation.html

```diff
@@ -0,0 +1,22 @@
+{{define "email/invite"}}
+{{- /* Message Header */ -}}
+Subject: COVID-19 Verification Server Invitation
+To: {{.ToEmail}}
+From: {{.FromEmail}}
+MIME-Version: 1.0
+Content-Type: text/html; charset="utf-8"
```

Comment:
If we set this as `text/plain`, you can remove the `<br>` and the email will "just work".